contracts/l2/tokens/static_a_token.cairo

```diff
@@ -212,31 +212,31 @@ end
 func push_rewards_index{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
     block_number : Uint256, rewards_index : Ray
 ):
-    claimable_push_rewards_index(block_number, rewards_index)
+    incentivized_erc20_push_rewards_index(block_number, rewards_index)
     return ()
 end
 
 @external
 func get_rewards_index{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
     rewards_index : Ray
 ):
-    let (res) = claimable_get_rewards_index()
+    let (res) = incentivized_erc20_get_rewards_index()
     return (res)
 end
 
 @external
 func get_user_rewards_index{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
     user : felt
 ) -> (user_rewards_index : Uint256):
-    let (res) = claimable_get_user_rewards_index(user)
+    let (res) = incentivized_erc20_get_user_rewards_index(user)
     return (res.ray)
 end
 
 @view
 func get_user_claimable_rewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
     user : felt
-) -> (user_claimable_rewards : Uint256):
+) -> (user_incentivized_erc20_rewards : Uint256):
```

Comment:
let's keep `user_claimable_rewards`here otherwise we will have to update the `static_a_tokens.test.ts` to use `user_incentivized_erc20_rewards`when calling for the user claimable reward. (that's why the test is failing)

Reply:
Ah, yeah that was a search and replace gone rogue